README.md

```diff
@@ -13,3 +13,13 @@ network. It should have the following properties:
   to enable more efficient communication when possible.
 - **Documented.** The [list-methods](https://localhost:8993/direct) rpc
   method should list out all known methods.
+
+## Installing on different versions
+
+Currently there is support for 2.19 and 2.25+ (this may run on versions between, but has not been tested).
+In order to install a specific version into a running distribution, build or install these artifacts locally, then run:
+
+`feature:repo-add mvn:com.connexta.jsonrpc/jsonrpc-features/0.5-SNAPSHOT/xml/features`
+
+
+This will add the two features for the previously mentioned features. Run `feature:install jsonrpc-2.19` or `feature:install jsonrpc-2.25` depending on which version you need.
```

Comment:
✏️ I don't understand "the two features for the previously mentioned features"

Reply:
Indeed, It doesn’t make a whole lot of sense. Lets see, I think i’ll change it to something along the lines of:
> The `feature:repo-add` will add the features for jsonrpc available in your system. Run `Feature:install`......

Does that sound a little better?

Reply:
👍